beacon-chain/rpc/eth/validator/handlers_test.go

```diff
@@ -1738,6 +1738,7 @@ func TestGetProposerDuties(t *testing.T) {
 	t.Run("next epoch", func(t *testing.T) {
 		bs, err := transition.GenesisBeaconState(context.Background(), deposits, 0, eth1Data)
 		require.NoError(t, err, "Could not set up genesis state")
+		require.NoError(t, bs.SetSlot(params.BeaconConfig().SlotsPerEpoch))
```

Comment:
This was another test bug because the old `CommitteeAssignments` would mutate state slot. We should set the state slot here similar to what we did in the first test case